Scripts/data_handling.py

```diff
@@ -5,16 +5,19 @@
 class MatrixData:
     def __init__(self):
         # Here we should actually set up a link to data files
-        self.base_demand = numpy.arange(49).reshape(7, 7)
+        self.base_demand = numpy.arange(25).reshape(5, 5)
     
     def get_data(self, mode):
         return self.base_demand
 
+    def get_zone_numbers(self):
+        return [5, 6, 7, 2792, 16001]
```

Comment:
Are these numbers some references to EMME constants or..? They could be declared as const variables with human readable names.

Reply:
Added # TODO to make it clearer that this definition is just temporary